lib/sql/escape.go

```diff
@@ -10,15 +10,14 @@ import (
 )
 
 type NameArgs struct {
-	Escape   bool
 	DestKind constants.DestinationKind
 }
 
 // symbolsToEscape are additional keywords that we need to escape
 var symbolsToEscape = []string{":"}
 
 func EscapeNameIfNecessary(name string, uppercaseEscNames bool, args *NameArgs) string {
-	if args == nil || !args.Escape {
```

Comment:
Is this safe? What about callers that expect the column to not be escaped?

Reply:
`args.Escape` is always true.

Reply:
Cool

Reply:
Do you want to kill NameArgs and just pass DestKind?

Reply:
I thought about that, and that's what I originally wanted to do, but it's a little tricky because sometimes `args` is null and it's annoying to create a pointer to `DestinationKind`. Maybe I'll do that anyways since it's a bit safer.

Reply:
Tried again and it ends up being a bigger lift than I want to do right now.